lib/nano.js

```diff
@@ -553,6 +553,9 @@ module.exports = exports = nano = function dbScope(cfg) {
       }
       qs = qs || {};
```

Comment:
We can actually change this line to `qs = Object.assign({}, qs)`. Object assign does the same as `qs = qs || {};`

I'm wondering if we should have another name for it. Maybe even just qs1 or something.

Reply:
👍